content/designer/multimedia-and-animations.md

```diff
@@ -160,9 +160,9 @@ Optional ideas to assess knowledge:
 
 {% include excol.html type="middle" %}
 
-Discuss different types of moving content, including carousels and animations. Present methods to avoid distractions, seizures, physical reactions, and unpredictable keyboard focus management caused by moving content. For example, support for pausing, stopping, or hiding moving content, as well as minimizing blinking and flashing when possible.
+Discuss different types of moving content, including carousels and animations. Present methods to avoid seizures, physical reactions, distractions, and unpredictable keyboard focus management caused by moving content. For example, support for pausing, stopping, or hiding moving content.
```

Comment:
```suggestion
Discuss different types of moving content, including carousels and animations. Present methods to avoid seizures, physical reactions, distractions, and unpredictable keyboard focus management caused by moving content. For example, support for pausing, stopping, or hiding moving content, as well as minimizing blinking and flashing.
```